src/grt/src/GlobalRouter.cpp

```diff
@@ -3871,6 +3914,7 @@ void GlobalRouter::addDirtyNet(odb::dbNet* net)
 
 void GlobalRouter::updateDirtyRoutes()
 {
+  setVerbose(false);
```

Comment:
why?

Reply:
I added `setVerbose(false)` to help with debugging, this was already removed in the last commits.